### PR TITLE
Use both MLOperandDescriptor.dimensions and MLOperandDescriptor.shape

### LINKED ISSUE #273: Rename `MLOperandDescriptor.dimensions` to `MLOperandDescriptor.shape`
https://chromium-review.googlesource.com/c/chromium/src/+/5502631

https://github.com/webmachinelearning/webnn/pull/676

/cc @ibelem @Honry

### PR DESCRIPTION
**Pull request by Honry**
Fixed #273

### CODE REVIEW

**Comment by Honry** [comment]
@huningxin, @ibelem, PTAL, thanks!

**Comment by ibelem** [comment]
Thanks @Honry . What is our plan for such kind of breaking changes @huningxin , I meant do we need to add notes in footer of sample pages? E.g. "Known compatible Chromium version: 130.0.6723.4 due to [#676](https://github.com/webmachinelearning/webnn/pull/676) "

**Comment by Honry** [comment]
> Thanks @Honry . What is our plan for such kind of breaking changes @huningxin , I meant do we need to add notes in footer of sample pages? E.g. "Known compatible Chromium version: 130.0.6723.6 due to [#676](https://github.com/webmachinelearning/webnn/pull/676) "

Maybe we should keep both `dimensions` and `shape` for a while to avoid breaking.

**Comment by huningxin** [comment]
> Maybe we should keep both `dimensions` and `shape` for a while to avoid breaking.

sounds good. The Chromium impl would also support both `dimensions` and `shape` for a while for a smooth transition.

**Comment by Honry** [comment]
> > Maybe we should keep both `dimensions` and `shape` for a while to avoid breaking.
> 
> sounds good. The Chromium impl would also support both `dimensions` and `shape` for a while (a milestone) for a smooth transition.

Done, PTAL, thanks!

File: common/utils.js [hunk 1]
```diff
@@ -121,7 +121,8 @@ export async function buildConstantByNpy(builder, url, targetType = 'float32') {
     throw new Error(`Conversion from ${npArray.dataType} ` +
         `to ${targetType} is not supported.`);
   }
-  return builder.constant({dataType: type, dimensions}, typedArray);
+  return builder.constant(
+      {dataType: type, dimensions: shape, shape}, typedArray);
```
Comment (ibelem) [inline common/utils.js:125]: LGTM, thanks  @Honry

**Review by ibelem — APPROVED** [review]

**Pull request merged**